packages/core/src/constants.ts

```diff
@@ -28,7 +28,7 @@ export const OUTGOING_EVENTS = {
 
 export const INCOMING_EVENTS = {
   RequestEditorMode: 'requestEditorMode',
-  CompositionUpdated: 'componentTreeUpdated',
```

Comment:
[Comment for reviewer] Since the actual value hasn't changed here and this is only used by User Interface we don't need to mark the old name as deprecated here. Instead I am removing it straight away.